docs/quickstart.rst

```diff
@@ -138,6 +144,8 @@ These API Keys can be utilized by providing them in a .env file. A .env file is
     BING_API_KEY="My Bing maps key"
     CENSUS_KEY="My census key"
```

Comment:
General question for the environment variables: would it be easier for the user to provide a couple of functions that wrap a call to os.environ so that they don't have to bother with the environment file? They would simply have to call the function in the notebook set_bing_api_key("My bing maps key").

Reply:
Good question :-). Let's discuss live